packages/code-studio/src/main/AppInit.tsx

```diff
@@ -72,21 +73,30 @@ async function loadPlugins(): Promise<DeephavenPluginModuleMap> {
       `${import.meta.env.VITE_MODULE_PLUGINS_URL}/manifest.json`
     );
 
+    if (!Array.isArray(manifest.plugins)) {
+      throw new Error('Plugin manifest JSON does not contain plugins array');
+    }
+
     log.debug('Plugin manifest loaded:', manifest);
-    const pluginPromises = [];
+    const pluginPromises: Promise<unknown>[] = [];
     for (let i = 0; i < manifest.plugins.length; i += 1) {
-      const { name, main } = manifest.plugins[i];
+      const { main } = manifest.plugins[i];
       const pluginMainUrl = `${
         import.meta.env.VITE_MODULE_PLUGINS_URL
-      }/${name}/${main}`;
+      }/${main}`;
```

Comment:
Why are you making this a breaking change? Don't, I don't want the churn. See discussion we're having with Devin about packaging the JS with python for delivery, and perhaps that invalidates the manifest entirely, or changes the format there, but I'd rather the breaking changes be when that gets implemented instead of breaking it twice.

Reply:
Just put the manifest.json file in the js-plugins folder in the `plugins` folder

Reply:
Ya I thought of that halfway through and didn't go back and edit. Either way we need better documentation around it

Reply:
For production cases, yes the [core docs](https://deephaven.io/core/docs/how-to-guides/configuration/js-plugins/#quickstart) are best, and that extracts the files and generates the manifest.json. We're not updating that tooling.
For dev mode, yes we need to update the readme in deephaven-js-plugins, likely instruct to have it just in the plugins directory instead, and the name just being the location of the package.json.

With python wrapped stuff, we'll likely just get rid of the manifest.json entirely or have a new format.